### DISCUSSION TITLE
Mobil?

### DISCUSSION THREAD

**Discussion by Kuafor5555** [IBAX-io/go-ibax#1654] [Bug Submission]
Ibax site versiyonundan önce mobil versiyonu kurması gerekiyordu çünkü insanlar mobil versiyonu olunca kendini güvende hissediyor ve bu airdropun reklam
Olması için iyi bir başlangıç olabilirdi tsklr

**Comment by Eddyhunter07** [comment]
Mobil gelir

**Reply by Kuafor5555** [reply]
Buna saten inanıyorum  ilk önce mobil versiyonu ile başlasaydı daha iyi reklam yapılabilir di  ibx sağlam bi proje başarılarının devamını diliyorum
